cognee/eval_framework/evaluation/run_evaluation_module.py

```
@@ -13,6 +15,10 @@
 from cognee.modules.data.models.metrics_base import MetricsBase
 
 
+setup_logging(logger.INFO)
```

Ruff (0.8.2)

Undefined name `logger`

(F821)